src/Pages/LogIn/SignUp/SignUpScreen.js

```diff
@@ -0,0 +1,162 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import LoginTooltipIcon from "../LoginTooltipIcon";
+import CheckAgreeContainer from "../CheckAgreeContainer";
+import SignupCreateAccountBtn from "./SignupCreateAccountBtn";
+import LineContainer from "../LineContainer";
+import SignupLoginBtn from "./SignupLoginBtn";
+import "./SignUpScreen.scss";
+
+const check = {
+  privacyNotice: `Tesla 계정을 생성함으로써 Tesla의 개인 정보 보호 공지 사항 및 이용 약관을 이해하였으며 이에 동의합니다.`,
+  receiveUpdates: `Tesla의 업데이트를 수신하겠습니다.`,
+};
+
+class SignUpScreen extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: "",
+      password: "",
+      first_name: "",
+      last_name: "",
+      inputType: "password",
+      visible: "표시",
+      isChecked: false,
+    };
+  }
+
+  accountInfo = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  handlePassword = () => {
+    this.setState({
+      inputType: this.state.inputType === "password" ? "text" : "password",
+      visible: this.state.visible === "표시" ? "숨김" : "표시",
+    });
+  };
+
+  basicCondition = () => {
+    this.setState(
+      {
+        isChecked: this.state.isChecked === false ? true : false,
+      },
+      () => {
+        console.log(this.state.isChecked);
+      }
+    );
+  };
+
+  createAccount = (e) => {
+    e.preventDefault();
+    fetch("http://10.58.1.69:8000/user/signup", {
+      method: "POST",
+      body: JSON.stringify({
+        email: this.state.email,
+        password: this.state.password,
+        first_name: this.state.first_name,
+        last_name: this.state.last_name,
+      }),
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        console.log(res);
+        if (!this.state.last_name) {
+          alert("성에는 글자만 포함될 수 있습니다.");
+        } else if (!this.state.first_name) {
+          alert("이름에는 글자만 포함될 수 있습니다.");
+        } else if (!this.state.email) {
+          alert("유효한 이메일 주소를 입력해야 합니다.");
+        } else if (!this.state.password) {
+          alert("비밀번호가 필요합니다.");
+        } else if (this.state.isChecked === false) {
+          alert("계속하려면 개인 정보 공지 및 이용 약관에 동의해야 합니다.");
+        } else if (!this.state.email.includes("@")) {
+          alert("이메일 주소에 '@'를 포함해 주세요.");
```

Comment:
```suggestion
        if (!this.state.last_name)  alert("성에는 글자만 포함될 수 있습니다.");
        if (!this.state.first_name) alert("이름에는 글자만 포함될 수 있습니다.");
        if (!this.state.email) alert("유효한 이메일 주소를 입력해야 합니다.");
        if (!this.state.password) alert("비밀번호가 필요합니다.");
        if (!this.state.isChecked) alert("계속하려면 개인 정보 공지 및 이용 약관에 동의해야 합니다.");
        if (!this.state.email.includes("@")) { alert("이메일 주소에 '@'를 포함해 주세요.");
```
- 개인적으로 else if 문은 가독성이 떨어져서 선호하는 편은 아닙니다
- 뒤에 나오는 문장이 한줄일 경우 한줄로 쓰는게 더 좋다고 생각합니다

Reply:
훨씬 가독성이 좋아졌네요!! 
감사합니답